CHANGELOG.md

```diff
@@ -1,20 +1,31 @@
 faucet-pipeline-css version history
 ===================================
 
+
+v0.4.0
+------
+
+_TBD_
```

Comment:
This will need to be updated, of course. We might also want to update the Git commit date accordingly.